src/riak_kv_test_util.erl

```diff
@@ -265,8 +264,9 @@ dep_apps(Test, Extra) ->
                 application:set_env(riak_core, ring_state_dir, Test ++ "/ring"),
                 application:set_env(riak_core, platform_data_dir, Test ++ "/data"),
                 application:set_env(riak_core, handoff_port, 0), %% pick a random handoff port
-                {ok, Dir} = file:get_cwd(),
-                Dirs = [Dir ++ "/../deps/*/priv"],
+                %% @TODO this is wrong still
```

Comment:
some indication of why it's wrong might be helpful

Reply:
…erm…you got me. It's "wrong" in that it is a hack maybe?

Reply:
Adding a comment to draw attention to it doesn't detract from the P/R - don't pay this one any more attention on my account.